app/src/main/java/io/apicurio/registry/ui/UserInterfaceConfigProperties.java

```diff
@@ -8,34 +8,34 @@
 @Singleton
 public class UserInterfaceConfigProperties {
 
-    @ConfigProperty(name = "registry.ui.contextPath", defaultValue = "/")
+    @ConfigProperty(name = "apicurio.ui.contextPath", defaultValue = "/")
     @Info(category = "ui", description = "Context path of the UI", availableSince = "3.0.0")
     public String contextPath;
-    @ConfigProperty(name = "registry.ui.navPrefixPath", defaultValue = "/")
+    @ConfigProperty(name = "apicurio.ui.navPrefixPath", defaultValue = "/")
     @Info(category = "ui", description = "Navigation prefix for all UI paths", availableSince = "3.0.0")
     public String navPrefixPath;
-    @ConfigProperty(name = "registry.ui.docsUrl", defaultValue = "/docs/")
+    @ConfigProperty(name = "apicurio.ui.docsUrl", defaultValue = "/docs/")
     @Info(category = "ui", description = "URL of the Documentation component", availableSince = "3.0.0")
     public String docsUrl;
     
     
-    @ConfigProperty(name = "registry.auth.url.configured")
+    @ConfigProperty(name = "quarkus.oidc.auth-server-url")
     public String authOidcUrl;
-    @ConfigProperty(name = "registry.ui.auth.oidc.redirectUri", defaultValue = "/")
+    @ConfigProperty(name = "apicurio.ui.auth.oidc.redirect-uri", defaultValue = "/")
     @Info(category = "ui", description = "The OIDC redirectUri", availableSince = "3.0.0")
     public String authOidcRedirectUri;
-    @ConfigProperty(name = "registry.ui.auth.oidc.clientId", defaultValue = "apicurio-registry-ui")
+    @ConfigProperty(name = "apicurio.ui.auth.oidc.client-id", defaultValue = "apicurio-registry-ui")
     @Info(category = "ui", description = "The OIDC clientId", availableSince = "3.0.0")
     public String authOidcClientId;
 
     
-    @ConfigProperty(name = "registry.ui.features.readOnly", defaultValue = "false")
+    @ConfigProperty(name = "apicurio.ui.features.readOnly", defaultValue = "false")
```

Comment:
Do we want to consolidate usage of camelCase vs hyphen-case? 
- `apicurio.ui.features.readOnly` -> `APICURIO_UI_FEATURES_READONLY` to
- `apicurio.ui.features.read-only.enabled` -> `APICURIO_UI_FEATURES_READ_ONLY_ENABLED`

See https://github.com/eclipse/microprofile-config/blob/main/spec/src/main/asciidoc/configsources.asciidoc#environment-variables-mapping-rules